<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixtureEditDialog</class>
 <widget class="QDialog" name="FixtureEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>470</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Fixture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/New folder/ColorSource-PAR_72.jpg</normaloff>:/New folder/ColorSource-PAR_72.jpg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="LightLevels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="FixtureLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixtureNumberDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>9</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixtureNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixture Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FixtureName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LumenLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lumens</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lumens">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CandelaBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CandelaLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Candela</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Illuminance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="LuxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lux</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LuxSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FootCandleSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="FootcandleLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Foot Candles</string>
    </property>
   </widget>
   <widget class="QLabel" name="IlluminaceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Illuminance:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Distance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="DistFeetLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Feet</string>
    </property>
   </widget>
   <widget class="QLabel" name="DistMetersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Meters</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DistFeetSpinBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DistMetersSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="DistLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="NavigationButtons_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Diameter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="FieldSizeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field diameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FieldSizeFeetLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Feet</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="BeamSizeFeetSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QLabel" name="FieldSizeMetersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Meters</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="BeamSizeMetersSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="FieldAngle">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>370</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="FieldAngleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field angle (Degrees)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FieldAngleSpinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RemoveFixture">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Fixture</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>326</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Navigationbuttons">
    <item>
     <widget class="QPushButton" name="Last">
      <property name="text">
       <string>&lt;&lt; Last</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Next">
      <property name="text">
       <string>Next &gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="NavigationButtons">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FixtureName</tabstop>
  <tabstop>Lumens</tabstop>
  <tabstop>CandelaBox</tabstop>
  <tabstop>DistMetersSpinBox</tabstop>
  <tabstop>DistFeetSpinBox_2</tabstop>
  <tabstop>LuxSpinBox</tabstop>
  <tabstop>FootCandleSpinBox</tabstop>
  <tabstop>BeamSizeMetersSpinBox</tabstop>
  <tabstop>BeamSizeFeetSpinBox</tabstop>
  <tabstop>FieldAngleSpinBox</tabstop>
  <tabstop>RemoveFixture</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>NavigationButtons</sender>
   <signal>accepted()</signal>
   <receiver>FixtureEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NavigationButtons</sender>
   <signal>rejected()</signal>
   <receiver>FixtureEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
